sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Demo.Tracing/Program.cs

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Diagnostics.Metrics;
+using OpenTelemetry;
+using OpenTelemetry.Resources;
+using Microsoft.Extensions.Logging;
+using OpenTelemetry.Metrics;
+using OpenTelemetry.Trace;
+
+namespace Azure.Monitor.OpenTelemetry.Exporter.Demo.Tracing
+{
+    public class Program
+    {
+        private const string ConnectionString = "InstrumentationKey=00000000-0000-0000-0000-000000000000";
+
+        public static void Main()
+        {
+            GenerateTraces();
+            GenerateLogs();
+            GenerateMetrics();
+        }
+
+        private static void GenerateTraces()
+        {
+            ActivitySource source = new ActivitySource("MyCompany.MyProduct.MyLibrary");
```

Comment:
Do we want this to be a local variable or a static member? What's the best practice?

Reply:
Best practice is to use it as static member. This is of disposable type, but no other samples disposed this object.